test/cookbooks/python-webapp-test/recipes/default.rb

```diff
@@ -0,0 +1,48 @@
+#
+# Cookbook Name:: osl_application_python_test
+# Recipe:: default
+#
+# Copyright 2015, OSU Open Source Lab
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_recipe 'python-webapp'
```

Comment:
You don't need this to use the `python_webapp_common` resource. That becomes available as soon as you put `depends 'python-webapp'` in the metadata.

In fact, if that line disappears from the metadata, this won't save you.